schema/workflow.json

```diff
@@ -556,7 +557,26 @@
       },
       "additionalProperties": false,
       "required": [
-        "produceEventRef"
+        "name", "data"
+      ]
+    },
+    "consumeeventref": {
+      "type": "object",
+      "description": "Waits for an event",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Reference to the unique name of a 'consumed' event definition",
+          "pattern": "^[a-z0-9](-?[a-z0-9])*$"
+        },
+        "consumeEventTimeout": {
```

Comment:
Why not take the opportunity to rename that property to `timeout`?

Reply:
I agree, you should have answered my https://github.com/serverlessworkflow/specification/pull/826#discussion_r1505827411

Reply:
Sorry, I overlooked it!